services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -36,7 +36,7 @@
               {
                 "id": "contactName",
                 "type": "text",
-                "validation": "text",
+                "validation": "textOptional",
```

Comment:
Blocking: Switch this back to `text` because we don't want to change it to optional for real.

Reply:
good catch